code/drasil-lang/lib/Language/Drasil/Document/Core.hs

```diff
@@ -69,16 +69,13 @@ data RawContent =
   | EqnBlock ModelExpr                       -- ^ Block of Equations holds an expression.
   | DerivBlock Sentence [RawContent]         -- ^ Grants the ability to label a group of 'RawContent'.
   | Enumeration ListType                     -- ^ For enumerated lists.
-  | Defini DType [(Identifier, [Contents])]  -- ^ Defines something with a type, identifier, and 'Contents'.
+  | Defini DType [(Sentence, [Contents])]    -- ^ Defines something with a type, identifier, and 'Contents'.
```

Comment:
Oh, I'm not so such about this particular change. While `Identifier` doesn't need to be `String`, I think there are still some non-trivial restrictions on them that `Sentence` does not obey.

I think we're going to need to introduce more types that are intermediate between `String` and `Sentence`.

Reply:
Is this something that you want done in this PR or are you talking more generally?

Reply:
I don't want this PR to change this particular type to `Sentence`, that seems like too big a change.

Reply:
@ManvendraJani It seems like @JacquesCarette is suggesting still using the `Identifier` type, but defining it to be a subtype of `Sentence` (e.g., that enforces certain properties) and that this PR shouldn't be merged until that is taken care of